articles/tech-resources/enabling-multiple-custom-tracking-domains.md

```diff
@@ -65,7 +65,7 @@ You can also configure custom tracking domains by using the SparkPost REST API.
 
 ## Using a Tracking Domain
 
-Once a tracking domain has been associated with a verified sending domain, the appropriate tracking domain will be inserted into all links if click tracking is turned on. Note that click tracking is turned on by default in both SparkPost.com and SparkPost Elite, and you must specifically include the option to turn it off in either the transmissions options (REST), or the SMTP API (SMTP).
+Once a tracking domain has been associated with a verified sending domain, the appropriate tracking domain will be inserted into all links if click tracking is turned on. Note that click tracking is turned on by default in both SparkPost.com and SparkPost Enterprise, and you must specifically include the option to turn it off in either the transmissions options (REST), or the SMTP API (SMTP).
```

Comment:
`...click tracking is turned on by default.` - remove the product names since they both have the same settings.